features/admin.login-flow-builder.v1/providers/authentication-flow-provider.tsx

```diff
@@ -273,12 +277,15 @@ const AuthenticationFlowProvider = (props: PropsWithChildren<AuthenticationFlowP
     };
 
     const isAdaptiveAuthAvailable: boolean = useMemo(() => {
-        if (orgType === OrganizationType.SUBORGANIZATION) {
+        const sharedAppAdaptiveAuthDisabled: boolean =
+            featureConfig?.disabledFeatures?.includes(SHARED_APP_ADAPTIVE_AUTH_FEATURE_ID);
+
+        if (orgType === OrganizationType.SUBORGANIZATION && sharedAppAdaptiveAuthDisabled) {
             return false;
         }
 
         return isAdaptiveAuthenticationAvailable;
-    }, [ isAdaptiveAuthenticationAvailable, orgType ]);
+    }, [ featureConfig, isAdaptiveAuthenticationAvailable, orgType ]);
```

Comment:
```suggestion
    }, [ isAdaptiveAuthenticationAvailable, orgType ]);
```

This will be not needed when the const is moved out of the hook